runtime/astar/src/lib.rs

```diff
@@ -185,8 +188,15 @@ impl Contains<RuntimeCall> for BaseFilter {
         match call {
             // Filter permission-less assets creation/destroying
             RuntimeCall::Assets(method) => match method {
-                pallet_assets::Call::create { id, .. } => *id < u32::MAX.into(),
-                pallet_assets::Call::destroy { id, .. } => *id < u32::MAX.into(),
+                pallet_assets::Call::create { id, .. } => *id < (u32::MAX as AssetId).into(),
```

Comment:
Lets drop here comment about limitation for asset-id of custom user assets. It should be < u32::MAX to create custom assets, by design it prevent mixing custom and service assets.